examples/python/hloc_glomap/README.md

```diff
@@ -0,0 +1,24 @@
+<!--[metadata]
+title = "Hierarchical-Localization and GLOMAP"
+tags = ["2D", "3D", "COLMAP", "Pinhole camera", "Time series", "GLOMAP", ]
+source = "https://github.com/pablovela5620/hloc-glomap"
+thumbnail = "https://static.rerun.io/thumbnail/6a5b887927834a6ea3db9474ce1e843ecd28b3cf/480w.png"
+thumbnail_dimensions = [480, 342]
+-->
+
+https://vimeo.com/1037241347?autoplay=1&loop=1&autopause=0&background=1&muted=1&ratio=2802:1790
```

Comment:
Can you wrap that in whatever html video tag is necessary for the video to be embedded in the readme please? (assuming that's even possible, but surely...)

Reply:
Is this so that it will render in the github? as I understand it, this is how Ive been doing it thus far for all other examples so that they would render on rerun.io example site. 

For example
https://github.com/rerun-io/rerun/tree/main/examples/python/depth_compare
https://rerun.io/examples/generative-vision/depth_compare

I don't totally understand whats going on in the background on the rerun site, but from what I've tried before, if I put it in an iframe tag it still won't render on github

Reply:
You know better than me :+1: